src/pyramid/interfaces.py

```diff
@@ -233,6 +241,10 @@ def encode_content(encoding='gzip', lazy=False):
 
     headers = Attribute(""" The headers in a dictionary-like object """)
 
+    is_authenticated = Attribute(
```

Comment:
@mmerickel @stevepiercy Does it follow the naming conventions to use a verb here for a property?  It feels a bit weird to me, but there is no reason to make it a method (it relies on `authenticated_identity` which is itself a property that calls many methods).  `authenticated` alone doesn’t seem so good either.

Reply:
Are you struggling with the question, "What is authenticated by `is_authenticated`?" If so, then `thing_is_authenticated`. If not, can you elaborate on what you want to convey to the developer with the name of the property?

Reply:
The naming `is_authenticated` is what I was expecting.